service/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/mailgun/service/repo"
+	"net/http"
+	"time"
+)
+
+var (
```

Comment:
Any reason to put these at this scope?

Reply:
Your right we do not need them. I Was going down the path here to use the init() function to be able to load a config file. But that would have taken too much time. Ideally, the init would read that file and then update those vars at that scope like port. the DB conn shouldn't have been there